tools/sand.md

```diff
@@ -0,0 +1,9 @@
+---
+name: sand
+creator: Collin Fair
+link: http://sand.cpfx.ca
+---
+
+sand lets you remix Pebble apps with new colours, features, and more - no coding required.
+
+<img src="http://i.imgur.com/d7QAI5f.png"/>
```

Comment:
True, I had changed it because I thought Github was messing up the image - then I realized it was because I browse the web at 125% zoom.